<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Settings</class>
 <widget class="QDialog" name="Settings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>128</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Custom Game</string>
  </property>
  <widget class="QLineEdit" name="lineEdit_col">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>32</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_height">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>54</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;Height:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_row</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_width">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>32</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;Width:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_col</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_mines">
   <property name="geometry">
    <rect>
     <x>16</x>
     <y>76</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&amp;Mines:</string>
   </property>
   <property name="buddy">
    <cstring>lineEdit_num</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_row">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>54</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_num">
   <property name="geometry">
    <rect>
     <x>72</x>
     <y>76</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ok">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>33</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_cancel">
   <property name="geometry">
    <rect>
     <x>128</x>
     <y>72</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
